### PR TITLE
metaschema-java v0.11.0 update

### PR DESCRIPTION
**Pull request by david-waltermire-nist**
# Committer Notes

- A few adjustments for modifications made in the [metaschema-java](https://github.com/usnistgov/metaschema-java/compare/v0.10.0..ed9350d413da25afdd50b3fdd679292b47f7509b) API and OSCAL model changes.
- Updating version to 1.0.5.0.

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/oss-maven/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/oss-maven/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [ ] Have you updated all website and readme documentation affected by the changes you made?

### CODE REVIEW

**Comment by david-waltermire-nist** [comment]
This PR won't build properly until the following OSCAL PRs are merged and the `oscal` submodule is updated.

- usnistgov/OSCAL#1804
- usnistgov/OSCAL#1805
- usnistgov/OSCAL#1808
- usnistgov/OSCAL#1819

**Comment by david-waltermire-nist** [comment]
Confirmed this PR addresses #13.

**Comment by david-waltermire-nist** [comment]
@aj-stein-nist Would you please mark #18 as resolved by this PR?

**Comment by david-waltermire-nist** [comment]
Added [unit test](https://github.com/david-waltermire-nist/liboscal-java/blob/a62dfb069756c75a915d07308d3a01ed97402533/src/test/java/gov/nist/secauto/oscal/java/OscalBindingContextTest.java#L175) for #13.

**Comment by david-waltermire-nist** [comment]
@aj-stein-nist Would you please mark #3 as resolved too. I confirmed this is working using a manual round-trip using oscal-cli.

**Comment by aj-stein-nist** [comment]
> @aj-stein-nist Would you please mark #18 as resolved by this PR?

Done.

**Comment by aj-stein-nist** [comment]
> @aj-stein-nist Would you please mark #3 as resolved too. I confirmed this is working using a manual round-trip using oscal-cli.

Done.

File: src/test/java/gov/nist/secauto/oscal/java/OscalBindingContextTest.java [hunk 1]
```diff
@@ -164,9 +157,50 @@ void testCatalogXmlListItems(@TempDir Path tempDir) throws IOException {
...
-    ISerializer<Catalog> serializer = context.newSerializer(Format.XML, Catalog.class);
+    ISerializer<Catalog> serializer = bindingContext.newSerializer(Format.XML, Catalog.class);
+    serializer.serialize(catalog, out);
+
+    assertNotNull(bindingContext.loadCatalog(out));
+    // out.delete();
+  }
+
+  @Test
+  void testLoadCatalogTightLists() throws IOException, URISyntaxException {
```
Comment (aj-stein-nist) [inline src/test/java/gov/nist/secauto/oscal/java/OscalBindingContextTest.java:169]: Minor, non-blocking: I like the regression tests!

**Review by aj-stein-nist — APPROVED** [review]

**Pull request merged**